chart/compass/charts/external-services-mock/templates/tenant-dependencies-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if eq .Values.global.externalServicesMock.tenantDependenciesConfig.manage true }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ .Values.global.tenantFetcher.dependenciesConfig.secretName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+stringData:
+  keyConfig: |
+    {
+    "eu-1": "{\n \"credentials\": {\n \"xsappname\": \"xsappname1\"\n}\n}",
```

Comment:
How these xsappname values can be changed in production? Currently, there are two hardcoded xsappnames for eu-1 and eu-2.

Reply:
These are generated in production as follows:
1. Create destination instances for each landscape (in the lss scripts) and store the credentials in vault
2. The deploy Jenkins job takes the credentials from vault and creates a secret in the format `{ "region": { "credentials": .... } }`.
3. The destination fetcher mounts the secret and parses the xsappnames. 
4. Destinations callback returns the proper xsappname given the region (passed as a url parameter)